api/src/models/user.ts

```diff
@@ -1,11 +1,109 @@
-import mongoose, { Document } from 'mongoose';
+import mongoose, { Document, Schema } from 'mongoose';
 
-interface User extends Document<any> {
+import { IPitch } from './pitch';
+
+/**
+ * Basic interface for a User Schema.
+ */
+interface IUser extends Document<any> {
```

Comment:
Is there a reason why this interface is different from the User schema below?

Reply:
I forgot to update it as I was changing the schema, my bad